scrapers/base_scraper.py

```
@@ -173,31 +171,37 @@ def validate_title_and_year(
+                )
+                return False
 
-        if catalog_type == "series" and parsed_year:
+        if catalog_type == "series" and parsed_data.get("year"):
             # If end_year exists, check parsed_year is within the range; otherwise, check parsed_year >= metadata.year
             if (
                 metadata.end_year
-                and not (metadata.year <= parsed_year <= metadata.end_year)
-            ) or (not metadata.end_year and parsed_year < metadata.year):
+                and not (metadata.year <= parsed_data.get("year") <= metadata.end_year)
+            ) or (not metadata.end_year and parsed_data.get("year") < metadata.year):
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)